modular-docs-manual/content/topics/module_guidelines-assembly.adoc

```diff
@@ -27,7 +27,16 @@ Use the second level heading syntax for the Prerequisites section in the assembl
 [discrete]
 == Assembly Modules
 
-List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly.
+To include modules in an assembly, use the Asciidoc  link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include directive]. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. Use the `leveloffset` attribute to set the hierarchy of the module relative to the assembly, as shown in the following example:
+
+.Level offset for module files
+----
+file1.adoc[leveloffset=+1]
+file2.adoc[leveloffset=+2]
+file3.adoc[leveloffset=+3]
+----
+
+NOTE: All module and assembly titles must use the H1 heading designation (`=`).
```

Comment:
@emmurphy1 , @rkratky , @fbolton , @Preeticp , @mikemckiernan @yzimmerm , This was approved back in May, but we've had some shuffling, a suggestion from Emily which I've now incorporated, and a lot of time since then, so resurfacing once more. If you're all still okay with this addition, please approve and/or comment accordingly and we'll merge from there. So sorry to leave this in the dust. Fell off my radar entirely. Thanks!

Reply:
I do not feel strongly, but I don't think it needs to be a note.  I know this is minutiae, but I'm also wincing at the parens because the ISG suggests avoiding parens.  Maybe use "...heading designation, `=`."

Reply:
@mikemckiernan Thanks. I changed to this. That work?

>All module and assembly titles must use the H1 heading designation, such as `= My heading`.

Reply:
Looks good. Thanks.